confgenerator/confgenerator_test.go

```diff
@@ -264,6 +312,11 @@ func generateConfigs(pc platformConfig, testDir string) (got map[string]string,
 	if err != nil {
 		return
 	}
+
+	if pc.otelLoggingEnabled {
+		mergedUc.Logging.Service.OTelLogging = true
```

Comment:
My intent was that we would have specific otel logging tests that already have a checked-in config file with `OtelLogging = true` (and an `EXPERIMENTAL_FEATURES` file to enable the experiment).

Do we really need to special-case it here? We don't do that for any of our other global flags.

Reply:
> Do we really need to special-case it here?

With this specific PR proposed idea of adding `linux-otel-logging` and `windows-otel-logging` folders to all tests, there has to be a way to enable both the `EXPERIMENTAL_FEATURES` env variable and the `cofig.yaml (experimental_otel_logging: true)`. So `mergedUc.Logging.Service.OTelLogging = true` is indeed needed.

We could modify `config.yaml` files to have this setting too, thought that maybe more complicated. I'll explore the solutions with all this feedback. Thanks for the comment!